src/handlebar-helpers.js

```diff
@@ -0,0 +1,26 @@
+// list of useful handlebars helpers we can bundle
+// credit goes to https://stackoverflow.com/a/31632215 & https://stackoverflow.com/a/50427569
+//
+// TODO: maybe make this extensible via another CR field?
+
+const objectPath = require('object-path');
+
+const helpers = {
+  assign: function (varName, varValue, options) {
+    objectPath.set(options, ['data', 'root', varName], varValue);
```

Comment:
is it possible for varName to ever not be a string? if it can be a number, you will end up with root being an array instead of an object

Reply:
oh good catch, that's sneaky

fixed in https://github.com/razee-io/MustacheTemplate/pull/137/commits/ba367a00ab8fc9c5ae5588c0d92e6d6d698367c4